com.unity.render-pipelines.high-definition/Editor/Material/UIBlocks/LayerListUIBlock.cs

```diff
@@ -69,19 +69,22 @@ protected override void OnGUIOpen()
             float resetButtonWidth = 43;
             float padding = 4f;
             float endOffset = 2f;
-            float labelWidth = 100f;
+            float labelWidth = EditorGUIUtility.labelWidth;
             float height = EditorGUIUtility.singleLineHeight + EditorGUIUtility.standardVerticalSpacing;
 
             EditorGUIUtility.labelWidth = labelWidth;
 
-            Rect headerLineRect = GUILayoutUtility.GetRect(1, height);
-            Rect headerLabelRect = new Rect(headerLineRect.x, headerLineRect.y, EditorGUIUtility.labelWidth - indentOffset + 15f, height);
-            Rect headerUVRect = new Rect(headerLineRect.x + headerLineRect.width - 37f - resetButtonWidth - endOffset, headerLineRect.y, UVWidth + 5, height);
-            Rect headerMaterialDropRect = new Rect(headerLineRect.x + headerLabelRect.width - 20f, headerLineRect.y, headerLineRect.width - headerLabelRect.width - headerUVRect.width, height);
+            Rect headerLineRect = GUILayoutUtility.GetRect(1, 15);
```

Comment:
can you add a const variable with the value `15`? in this way we can easily change it.